saleor/graphql/checkout/tests/test_checkout_complete.py

```diff
@@ -889,7 +897,7 @@ def test_checkout_complete_insufficient_stock_payment_voided(
     assert data["errors"][0]["message"] == "Insufficient product stock: 123"
     assert orders_count == Order.objects.count()
 
-    gateway_void_mock.assert_called_once_with(
+    gateway_refund_mock.assert_called_once_with(
```

Comment:
The name of the tests has string - `payment_voided`, but you're calling assert for refund. Not sure if this is expected.

Reply:
I've managed to find a way to make the dummy gateway only authorize the payment, rather than fully charging. The test is now calling void once again.